tests/layer_tests/pytorch_tests/test_bucketize.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestBucketize(PytorchLayerTest):
+
+    def _prepare_input(self, input_shape, boundaries_range, input_dtype, boundaries_dtype, out_int32):
+        if out_int32:
+            output_dtype = "int32"
+        else:
+            output_dtype = "int64"
+        return (
+            np.random.randn(*input_shape).astype(input_dtype), 
+            np.arange(*boundaries_range).astype(boundaries_dtype),
+            np.zeros(input_shape).astype(output_dtype))
+
+    def create_model(self, out_int32, right, is_out):
+        class aten_bucketize(torch.nn.Module):
+
+            def __init__(self, out_int32, right, is_out) -> None:
+                super().__init__()
+                self.out_int32 = out_int32
+                self.right = right 
+                self.is_out = is_out
+
+            def forward(self, input, boundaries, output):
```

Comment:
Please create `output` inside `forward` and return it with bucketize result, so it can be validated

Reply:
You now return `output`, but please remove `output` from inputs of `forward` and create it inside with `zeros_like` or `emply_like`.

Reply:
Thank you for your advice! I have incorporated your suggestions.